definitions/npm/bser_v1.x.x/flow_v0.22.x-/bser_v1.x.x.js

```diff
@@ -0,0 +1,58 @@
+declare type $npm$bser$BserTypes =
```

Comment:
Awesome stuff! Just one question: Why does this type needs to be global? I can't see any other file / code part except the `bser` module using it...?